### DISCUSSION TITLE
Getting Metadata XML

### DISCUSSION THREAD

**Discussion by ChadwickLloyd** [codegreencreative/laravel-samlidp#100] [Q&A] [answered]
Linked to: https://github.com/codegreencreative/laravel-samlidp/issues/97

Hi all,

I'm aiming to use a moodle site as a service provider, where a user can login to a laravel web app, then access moodle having been verified.

To set up moodle as a SP, it requires some setup steps (at least with the plugin I have found). One of these steps is to input a metadata xml.

So far I am unable to find a generated Metadata xml file, or find generated xml.

Is there some way to do this? or any advice on setting up moodle as a SP.

any help will be appreciated.

Cheers.

**Comment by upwebdesign** [comment, marked as answer]
@ChadwickLloyd you can find the metadata URL at https://yourdomain.com/saml/metadata

**Reply by ChadwickLloyd** [reply]
thankyou! I appreciate the support
